test/recipes/aws-parallelcluster-install/controls/sudoers.rb

```diff
@@ -0,0 +1,24 @@
+# Copyright:: 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could you please put this control in `test/recipes/platform_setup/controls` in order to be consistent with the structure we agreed upon?

Reply:
My idea is to keep the structure aligned with the structure of the cookbook and do this refactoring using functional area names in future.